shinkai-libs/shinkai-sqlite/src/lib.rs

```diff
@@ -116,43 +131,67 @@ impl SqliteManager {
         Ok(())
     }
 
+    fn initialize_agents_table(conn: &rusqlite::Connection) -> Result<()> {
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS shinkai_agents (
+                agent_id TEXT NOT NULL UNIQUE,
+                name TEXT NOT NULL,
+                full_identity_name TEXT NOT NULL, -- Store as a JSON string
+                llm_provider_id TEXT NOT NULL,
+                ui_description TEXT NOT NULL,
+                knowledge TEXT NOT NULL,
+                storage_path TEXT NOT NULL,
+                tools TEXT NOT NULL,
+                debug_mode INTEGER NOT NULL,
+                config TEXT -- Store as a JSON string
+            );",
+            [],
+        )?;
+
+        Ok(())
+    }
+
     fn initialize_my_subscriptions_table(conn: &rusqlite::Connection) -> Result<()> {
         conn.execute(
             "CREATE TABLE IF NOT EXISTS my_subscriptions (
                 subscription_id TEXT NOT NULL UNIQUE,
+                subscription_id_data BLOB NOT NULL,
                 shared_folder TEXT NOT NULL,
-                streaming_node TEXT NOT NULL,
+                streaming_node TEXT NOT NULL, -- Store as a JSON string
```

Comment:
streaming node is a shinkainame we shouldnt save it as a json string it's just an identity name eg. @@nico.shinkai so it should just be TEXT